core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -256,9 +276,22 @@ def detect_frontend_to_install
         (options[:auto_accept] && DEFAULT_FRONTEND) ||
         ask(<<~MSG.indent(8), default: DEFAULT_FRONTEND, limited_to: FRONTENDS.keys)
 
-          Which frontend would you like to use? solidus_starter_frontend is
-          recommended. However, some extensions are still only compatible with
-          the now deprecated solidus_frontend.
+          Which frontend would you like to use?
+          Selecting #{DEFAULT_FRONTEND} is recommended.
+          However, some extensions are still only compatible with the now deprecated #{LEGACY_FRONTEND}.
+
+        MSG
+    end
+
+    def detect_authentication_to_install
+      ENV['AUTHENTICATION'] ||
+        options[:authentication] ||
+        (Bundler.locked_gems.dependencies['solidus_auth_devise'] && 'solidus_auth_devise') ||
```

Comment:
Should we use `DEFAULT_AUTHENTICATION` instead of `'solidus_auth_devise'` in this line?

Reply:
Done thanks! 👍